tests/Feature/GoalTest.php

```diff
@@ -45,4 +45,36 @@ public function test_goal_tracks_some_transactions()
                 'amount' => 100,
             ]);
     }
+
+    public function test_goal_require_a_name()
```

Comment:
Good start, but as you can see validation tests may become very hard to maintain. DataProvider can help us reduce the duplication and test all scenarios in one test method. Please check this article: https://medium.com/@daaaan/a-guide-to-unit-testing-laravel-form-requests-in-a-different-way-f1bdb6d86053